lib/irb/source_finder.rb

```diff
@@ -18,6 +18,9 @@ def initialize(irb_context)
 
     def find_source(signature)
       context_binding = @irb_context.workspace.binding
+      if signature.kind_of?(Array)
+        signature, s_count = signature
+      end
```

Comment:
Can this be a separate method parameter?
```ruby
def find_source(signature, s_count = nil)
```

Reply:
Done!